### PR TITLE
koord-scheduler: modify cpu and memory quantity formats and scales

### PR DESCRIPTION
**Pull request by xulinfei1996**
Signed-off-by: xulinfei.xlf <xulinfei.xlf@alibaba-inc.com>

### Ⅰ. Describe what this PR does
Change pod's cpu quantity to MilliQuantity in order to calculate runtimeQuota correctly, change memory's format from DecimalSI to BinarySI, fix UT.

### V. Checklist

- [ ] I have written necessary docs and comments
- [ ] I have added necessary unit tests and integration tests
- [ ] All checks passed in `make test`

### CODE REVIEW

File: pkg/scheduler/plugins/elasticquota/core/runtime_quota_calculator.go [hunk 1]
```diff
@@ -418,3 +418,23 @@ func (qtw *RuntimeQuotaCalculator) logQuotaInfoNoLock(verb string, quotaInfo *Qu
 		quotaInfo.CalculateInfo.Runtime, quotaInfo.CalculateInfo.Used, qtw.treeName, qtw.totalResource, qtw.groupReqLimit,
 		qtw.globalRuntimeVersion)
 }
+
+func getValue(res resource.Quantity, resName v1.ResourceName) int64 {
```
Comment (eahydra) [inline pkg/scheduler/plugins/elasticquota/core/runtime_quota_calculator.go:422]: make the name more meaningful. maybe getQuantityValue is better?

**Review by eahydra — CHANGES_REQUESTED** [review]

File: pkg/scheduler/plugins/elasticquota/core/runtime_quota_calculator.go (hunk 1 above)
Comment (xulinfei1996) [reply pkg/scheduler/plugins/elasticquota/core/runtime_quota_calculator.go]: fixed

File: pkg/scheduler/plugins/elasticquota/plugin_test.go [hunk 2]
```diff
@@ -584,6 +584,13 @@ func createResourceList(cpu, mem int64) corev1.ResourceList {
 	}
 }
 
+func createResourceList2(cpu, mem int64) corev1.ResourceList {
```
Comment (jasonliu747) [inline pkg/scheduler/plugins/elasticquota/plugin_test.go:587]: is `createResourceList` needed after this change?

Comment (xulinfei1996) [reply pkg/scheduler/plugins/elasticquota/plugin_test.go]: fixed, modify the createResourceList2 function as createResourceList, then delete the function createResourceList2

Comment (jasonliu747) [reply pkg/scheduler/plugins/elasticquota/plugin_test.go]: image

Comment (xulinfei1996) [reply pkg/scheduler/plugins/elasticquota/plugin_test.go]: createResourceList2是没有倍数，而createResourceList是*1000的。

**Review by buptcozy — COMMENTED** [review]
/lgtm

**Review by buptcozy — APPROVED** [review]
/lgtm

**Review by eahydra — APPROVED** [review]
/lgtm

**Pull request merged**